pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
```

Comment:
> So how does "device" play in here. Is it a single zram "partition" / "disk" ?

Yep. When enabling zram, you can tell it how many virtual disks (called devices) to create.